scripts/custom/clean_osiris_codes.py

```diff
@@ -0,0 +1,9 @@
+# function clean_osiris_codes
+# remove all chars before and including the first colon
+
+
+def clean_osiris_codes(raw_input):
+    if raw_input is None:
```

Comment:
Here you can use the is_empty function defined in utils,
`if utils.is_empty(raw_input):`
 which covers all cases where the raw input can be empty. You can look at other cleaning scripts for guidance, like  cutsom/clean_date.py